docs/docs/statistics/power.mdx

```diff
@@ -116,14 +124,14 @@ Frequently asked questions:
 3. What if my experiment is low-powered? Should I still run it? The biggest cost to running a low-powered experiment is that your results will probably be noisy, and you will face ambiguity in your rollout/rollback decision. That being said, you will probably still have learnings from your experiment.
 4. What does "N/A" mean for my MDE result? It means there is no solution for the MDE, given the current number of weekly users, control mean, and control variance. Increase your number of weekly users or reduce your number of treatment variations.
 
-## GrowthBook implementation
+## GrowthBook implementation (frequentist)
```

Comment:
At this point, this documentation is really long and complicated and I'm worried about having it be public facing when so few people will find it relevant, and the few people who are technical and are curious about our approach cannot parse this information effectively.

Can we reformat and simplify as much as possible?

Some suggestions:
* Clear formulae for each object that we care about (Power = ...; MDE = ...)
* Plain language summary of our approach?
* Minimal recreation of concepts explained elsewhere (e.g. var(\Delta))
* Less explanation of the steps to solve the math, and a bit more reliance on english to explain how we ended up with final result. We can keep more documentation in an internal notion doc.
* More sign posting and better formatting. This document is a wall of uneven line breaks and interspersed equations and plain text. Can we consider using more sub-headers, bolded concepts like "Power" with their equation right below it?

This comment is asking for a pretty big overhaul but this section is a bit messy and I think doesn't do a lot to build trust in the hard work you've done to do this right.

This applies to both Freq and Bayesian power sections.